scripts/aem.js

```diff
@@ -1,3 +1,18 @@
+/**
+ * Sanitizes a string for use as class name.
+ * @param {string} name The unsanitized string
+ * @returns {string} The class name
+ */
+function toClassName(name) {
```

Comment:
Is that for a CSS or JS class name? I'd call it either `toCssClassName` or `toJsClassName` for clarity.

Reply:
`toClassName` is by now a [well known function in franklin ](https://github.com/adobe/aem-boilerplate/blob/efc82141de8092949e8ea3354649966cf200c2d8/scripts/aem.js#L175) so I would suggest leaving the name as-is